Directory.Build.props

```diff
@@ -13,5 +13,17 @@
     <TargetsWindows Condition="$(RuntimeIdentifier.StartsWith('win'))">true</TargetsWindows>
     <TargetsLinux Condition="$(RuntimeIdentifier.StartsWith('linux'))">true</TargetsLinux>
     <LangVersion>Latest</LangVersion>
+
+    <!-- Packaging props -->
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <!-- We need to clear PackageLicenseUrl since arcade sets it, and since we're using the new compilers
```

Comment:
nit: Compilers should have nothing to do with NuGet so this comment may be a little unusual to future folks outside the context of this change.  I'd merely state that the property is deprecated so we ensure it isn't set.

Reply:
Ok, sounds good.